cannot have multiple `_` parameters in function

<!-- Please make sure to run `v up` before reporting any issues as it may have already been fixed.
     It's also advisable to update all relvant modules using `v outdated` and `v install` -->

<!-- You can use `v doctor` to fill up the next fields -->
**V version:** V 0.2.2 7eb7d04
**OS:** linux, Ubuntu 20.04.1 LTS (WSL 2)

**What did you do?**
```v
fn my_fn(_ int, _ f32) { println('I don\'t use those pesky params') }
my_fn(1, 1.2)
```

**What did you expect to see?**
> I don't use those pesky params
 
**What did you see instead?**
```
==================
      |                                  ~~~~^
/tmp/v/fn.8540435567263175054.tmp.c:8472:45: error: conflicting types for ‘_’
 8472 | VV_LOCAL_SYMBOL void main__my_fn(int _, f32 _) {
      |                                         ~~~~^
/tmp/v/fn.8540435567263175054.tmp.c:8472:38: note: previous definition of ‘_’ was here
 8472 | VV_LOCAL_SYMBOL void main__my_fn(int _, f32 _) {
      |                                  ~~~~^
...
==================
(Use `v -cg` to print the entire error message)

builder error:
==================
C error. This should never happen.

If you were not working with C interop, please raise an issue on GitHub:

https://github.com/vlang/v/issues/new/choose

You can also use #help on Discord: https://discord.gg/vlang
```

**Rationale**
Some implementation of the same method (when implementing an interface for example) may or may not use all the parameters. There are examples in vlib too, where an OS would use all params, and another only some of them.
So the solution isn't to just forbid using `_` in parameters in my opinion.

--- a/vlib/v/gen/fn.v
+++ b/vlib/v/gen/fn.v
@@ -239,7 +239,7 @@ fn (mut g Gen) fn_args(args []table.Param, is_variadic bool) ([]string, []string
 	mut fargs := []string{}
 	mut fargtypes := []string{}
 	for i, arg in args {
-		caname := c_name(arg.name)
+		caname := if arg.name == '_' { g.new_tmp_var() } else { c_name(arg.name) }
 		typ := g.unwrap_generic(arg.typ)
 		arg_type_sym := g.table.get_type_symbol(typ)
 		mut arg_type_name := g.typ(typ) // util.no_dots(arg_type_sym.name)

--- a/vlib/v/tests/blank_ident_test.v
+++ b/vlib/v/tests/blank_ident_test.v
@@ -10,6 +10,20 @@ fn test_fn_with_blank_param() {
 	fn_with_blank_param(321)
 }
 
+fn fn_with_multiple_blank_param(_ int, _ f32) {
+	_ = 'not an int nor a float'
+}
+
+struct Abc {}
+
+fn (_ Abc) fn_with_multiple_blank_param(_ int, _ f32) {}
+
+fn test_fn_with_multiple_blank_param() {
+	fn_with_multiple_blank_param(1, 1.1)
+	a := Abc{}
+	a.fn_with_multiple_blank_param(1, 1.1)
+}
+
 fn test_for_in_range() {
 	for _ in 1 .. 10 {
 		assert true

